fly/commands/unpause_job.go

```diff
@@ -22,16 +24,20 @@ func (command *UnpauseJobCommand) Execute(args []string) error {
 		return err
 	}
 
-	found, err := target.Team().UnpauseJob(command.Job.PipelineName, command.Job.JobName)
+	team := command.TeamFlag.GetTeamTarget(target)
+	found, err := team.UnpauseJob(pipelineName, jobName)
 	if err != nil {
 		return err
 	}
 
 	if !found {
-		return fmt.Errorf("%s/%s not found\n", command.Job.PipelineName, command.Job.JobName)
+		if command.Team == "" {
+			fmt.Println("hint: are you missing '--team' to specify the team for the build?")
```

Comment:
(same feedback as before; maybe just put the team in the error message)

Reply:
umm, after adding the team into the 404 message, I found it really confusing to see it. not sure if they can take it as a hint when debugging tho. 😛

Reply:
I think it makes sense to combine the error messages like @vito said. Right now, this error message would always be printed, even if the user is only part of one team. Combining the two messages makes sense and gives the user all the info they'd need to debug why their command failed.

Currently we'd see this:
```
% fly -t local unpause-job -j some-pipeline/some-job
hint: are you missing '--team' to specify the team for the build?
error: some-pipeline/some-job not found
```

If we combine it:
```
% fly -t local unpause-job -j some-pipeline/some-job
error: some-pipeline/some-job not found on team some-team
```

Reply:
Thanks! I already updated the code. Are you still seeing that output?